### PR TITLE
update middle section

### PR DESCRIPTION
**Pull request by fatmahussein**
## Portfolio : finish mobile version

Add:
 - Works section
 - About myself section

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @fatmahussein 
I'm sure you put a lot of effort into this project but this PR is invalid because you have unedited README file😓 please use this [template](https://github.com/microverseinc/readme-template)

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by huseyinbiyik** [comment]
Hi @fatmahussein,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

- You have submitted a project, that has no license file. Kindly add a license file, MIT is recommended, and submit for review again.👍
- There is a conflict on this branch please fix that as well.🙃

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -39,29 +41,197 @@
...
+             <a id="proj" href="#" class="button">See Project</a>
+            </div>
+
+            </div>
+            </div>
+          </div>
+        </div>
+       <!-- <div class="grid-item">2</div>
+        <div class="grid-item">3</div>  
+        <div class="grid-item">4</div>-->
```
Comment (Tufahel) [inline index.html:181]: - [ ] Kindly remove the `comments` from the code-base so that your code looks clean and readable.

File: styles.css [hunk 2]
```diff
@@ -286,3 +295,609 @@ h1 {
...
+  flex-direction: column;
+  position: absolute;
+  height: 1263px;
+  top: 3590px;
+  margin: 0;
+  border-top-right-radius: 50px;
+
+  /* Neutrals/N0 */
+  background: #fff;
+}
```
Comment (Tufahel) [inline styles.css:632]: - [ ] When I increased the size of the screen a bit from `400px to 728px`, the design inside the `about me` section collapsed. `Frameworks` and `Skills` override the other designs. The design should be consistent with all the mobile screen sizes. Kindly fix this.
  ezgif com-gif-maker (3)

File: index.html [hunk 3]
```diff
@@ -39,29 +41,197 @@
...
-    </section>  
   </div>
+   <div class="item" id="projects">
+    <a id="proj" href="#" class="button">Get my resume</a>
+   </div>
+</div>
+
+
+        <div class="Language-list" >
+            <h4 id="lang">Languages</h4>
```
Comment (Tufahel) [inline index.html:210]: - [ ] As per the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), there should be a `down arrow` on the right side of `Languages`. Kindly fix this to match the design.
  - ***Your Design***
    Screenshot from 2022-12-10 01-11-02

  - ***Figma Design***
  Screenshot from 2022-12-10 01-13-54

File: styles.css [hunk 4]
```diff
@@ -286,3 +295,609 @@ h1 {
...
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  position: absolute;
+  width: 375px;
+  height: 2788px;
+  left: 0;
+  top: 718px;
+}
```
Comment (Tufahel) [inline styles.css:310]: - [ ] Cards/Grids should be in the `middle` for all `mobile screens`. But, when I increase the size of the screen a bit from `400px to 728px` the cards move to the `left side` of the screen. Kindly fix this.

  ezgif com-gif-maker (4)

File: styles.css [hunk 5]
```diff
@@ -286,3 +295,609 @@ h1 {
...
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 203px;
+  height: 24px;
+  margin-top: 0%;
+}
+
```
Comment (Tufahel) [inline styles.css:468]: - [ ] The space from `text to tags` and `tags to button (See Project)` should be the same as per the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Kindly fix this to match the design.
  - ***Your Design***
  Screenshot from 2022-12-10 01-40-53

  - ***Figma Design**
  Screenshot from 2022-12-10 01-45-40

File: index.html [hunk 6]
```diff
@@ -39,29 +41,197 @@
...
+                <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <ul class="tags">
+               <li id="html"><a href="#" class="tags button1">html</a></li>
+               <li id="css"><a href="#" class="tags button2">css</a></li>
+               <li id="js"><a href="#" class="tags button3">javascript</a></li>
+               </ul> 
+             
+               <div class="project">
+                <a id="proj" href="#" class="button" >See Project</a>
+               </div>
```
Comment (Tufahel) [inline index.html:91]: - [ ] Great Work :+1:  on implementing `click` interactions on `button` for `See Project` and `Get my Resume`. Kindly implement other button interactions for those `buttons` to match the project requirements.
  - ***Requirement***
  Screenshot from 2022-12-10 00-08-39

  - You can consider adding following styles:

```
/* Enabled */
color: #6070FF;
background: #FFFFFF;

/* Hover */
color: #FFFFFF;
background: #6070FF;

/* Pressed */
color: #FFFFFF;
background: #2230D2;

/* Disabled */
color: #5E6C84;
background: #FFFFFF;
```

Screenshot from 2022-12-10 00-07-00

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @fatmahussein, :wave: 

Good Work So Far!!! :+1:  
You are almost done. There are some issues you need to fix before you move to the next project.

## Highlights 🌟

Alt Text
✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.

## Required Changes ♻️

- [ ] Kindly `rename` the file `LICENCE.md` as `MIT.md`, so that it can be consistent with `MIT` link of Readme file.
  Screenshot from 2022-12-10 01-26-43

- [ ] I can see many comments in the `CSS` file. Kindly remove the comments to make your code clean and readable.


_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 7]
```diff
@@ -281,8 +270,542 @@ h1 {
...
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  position: absolute;
+  width: 375px;
+  height: 2788px;
+  left: 0;
+  top: 718px;
+}
```
Comment (jerryowusu) [inline styles.css:288]: As requested by the previous reviewer, the cards should be in the middle of the screen, but it shifts to the left side when you stretch and increase the width of the screen. Kindly check and ensure that it has even margins on both left and right and does not shift to the side. :pray: 

Screenshot from 2022-12-10 15-02-17

File: README.md [hunk 8]
```diff
@@ -16,10 +16,17 @@
 - [❓ FAQ](#faq)
 - [📝 License](#license)
 
+
+# 📖 [Portfolio: finish mobile version] <a name="Hello-World"></a>
+
+
+**[Portfolio: finish mobile version]** is a project that shows you how to create your portfolio as a developer.
+=======
```
Comment (jerryowusu) [inline README.md:24]: [OPTIONAL] Your readme file is missing key headers/sections like the `Key Features`  and `Future Features`. Kindly ensure that all sections of the microverse readme template and present. :wink:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @fatmahussein 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost-there-reach


## Highlights
All linters are passing :heavy_check_mark: 
Neat and well organised code :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 8 above)
Comment (Qoosim) [inline README.md:24]: [OPTIONAL] Your readme file is missing key headers/sections like the Key Features and Future Features. Kindly ensure that all sections of the microverse readme template and present. wink

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @fatmahussein, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**Kindly check an optional comment I left under the review and please work on it**_ :pray: 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**